salt/metalk8s/salt/master/deployed.sls

```diff
@@ -31,3 +31,12 @@ Install and start salt master manifest:
       salt_master_version: {{ salt_master_version }}
     - require:
       - file: Create salt master directories
+
+Make sure salt master container is up:
```

Comment:
Shouldn't this command require "Install and start salt master manifest" ?

Reply:
right